.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+<!--
+Thanks for contributing to analog.now!
+Please follow the template below.
+Keep it clear and concise. Remove any section that doesn’t apply.
+-->
+
+## Description
+
+Briefly describe what you did and why.
+
+## Screenshots / Recordings
+
+Add screenshots or recordings here to help reviewers understand your changes.
+
+## Type of Change
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Breaking change
+- [ ] UI/UX update
+- [ ] Docs update
+- [ ] Refactor / Cleanup
+
+## Related Areas
+
+- [ ] Authentication
+- [ ] Calendar UI
+- [ ] Data/API
+- [ ] Docs
+- [ ] Tests
+
+## Testing
+
+- [ ] Added/updated tests
+- [ ] Manual testing performed
+- [ ] Cross-browser testing (if UI changes)
+- [ ] Mobile responsiveness verified (if UI changes)
+
+## Checklist
+
+- [ ] I’ve read the [CONTRIBUTING](../CONTRIBUTING.md) guide
+- [ ] I have run the linting tool
```

Comment:
We have both `bun run lint` and `bun run format`, these are both tested for PRs using a GitHub Action.